src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx

```diff
@@ -44,13 +44,9 @@ export const TransactionDetailsHashAndActionsSection: React.FC<Props> = ({ trans
 
   const onRetrySwap = useCallback(() => {
     Navigation.handleAction(Routes.WALLET_SCREEN, {});
-    Navigation.handleAction(Routes.EXCHANGE_MODAL, {
-      params: {
-        meta: retrySwapMetadata,
-        inputAsset: retrySwapMetadata?.inputAsset,
-        outputAsset: retrySwapMetadata?.outputAsset,
-      },
-    });
+
+    // TODO: Add retry swap logic back for swaps
```

Comment:
need to do this still

Reply:
ticketed this out instead since it's a little more involved and touches raps: https://linear.app/rainbow/issue/APP-1927/bring-back-in-failedswaps-retry-logic